packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx

```diff
@@ -31,7 +31,7 @@ const TopCastDisplay = () => {
           p: 1,
         }}
       >
-        Top Cast
+        {`Top Cast ${movieID}`}
```

Comment:
👁️

Reply:
We will delete this in the next PR. Right now it showcases the idea and usage...